### PR TITLE
Des 243 switch text levels

### PR DESCRIPTION
**Pull request by alimpens**

### CODE REVIEW

File: packages/react/README.md [hunk 1]
```diff
@@ -16,8 +16,6 @@ TODO: add alpha, beta and production statuses to components. Also show in Storyb
 
 ## Getting started
 
-TODO: These packages haven't been published yet, so the following doesn't work at the moment. You can currently only use these instructions when locally linking the packages to a consumer project.
-
 Install the packages you need, for instance:
 
 `npm install @amsterdam/design-system-react @amsterdam/design-system-css @amsterdam/design-system-tokens @amsterdam/design-system-font`
```
Comment (dlnr) [inline packages/react/README.md:21]: Do they have to install tokens separately? I would've thought the tokens are compiled to css in the css package.

File: storybook/storybook-css/src/docs/typography.stories.mdx [hunk 2]
```diff
@@ -23,23 +23,23 @@ In de tabel hieronder vind je deze waardes in pixels:
 
 | niveau      | 0   | 1   | 2   | 3   | 4   | 5   | 6   | 7   |
 | ----------- | --- | --- | --- | --- | --- | --- | --- | --- |
-| min         | 32  | 32  | 28  | 24  | 22  | 20  | 18  | 16  |
-| breakpoint  | 64  | 48  | 36  | 29  | 26  | 23  | 21  | 17  |
-| max         | 80  | 56  | 40  | 32  | 28  | 25  | 22  | 18  |
-| line-height | 1.1 | 1.2 | 1.3 | 1.4 | 1.5 | 1.4 | 1.6 | 1.6 |
+| min         | 32  | 32  | 28  | 24  | 20  | 22  | 18  | 16  |
```
Comment (dlnr) [inline storybook/storybook-css/src/docs/typography.stories.mdx:26]: This is not correct is it? 5 larger then 4?

File: storybook/storybook-css/src/docs/typography.stories.mdx [hunk 3]
```diff
@@ -23,23 +23,23 @@ In de tabel hieronder vind je deze waardes in pixels:
...
+| level | element                              |
+| ----- | ------------------------------------ |
+| 0     | Hero                                 |
+| 1     | Heading level 1                      |
+| 2     | Heading level 2                      |
+| 3     | Heading level 3, Blockquote          |
+| 4     | Heading level 4, Toptask Button      |
+| 5     | Paragraph large                      |
+| 6     | Paragraph, alle Links behalve inList |
+| 7     | Paragraph small, inList Link         |
```
Comment (dlnr) [inline storybook/storybook-css/src/docs/typography.stories.mdx:42]: Maybe also name Icon? it has a text-level assigned right?

File: storybook/storybook-css/src/docs/typography.stories.mdx (hunk 2 above)
Comment (dlnr) [reply storybook/storybook-css/src/docs/typography.stories.mdx:26]: This is not confusing at all 😬

Comment (alimpens) [reply storybook/storybook-css/src/docs/typography.stories.mdx:26]: It still is a little bit, but [there's a ticket to look into that](https://gemeente-amsterdam.atlassian.net/browse/DES-244)

**Pull request merged**